### ISSUE TITLE
Cannot build for PyPy on Windows

### ISSUE THREAD

**Issue by tyrius02** [conda-forge/portalocker-feedstock#65]
### Comment:

Portalocker on Windows depends on pywin32.win32con. Unfortunately the PyPy package of pywin32 does not include this module, causing the build to fail.

**Comment by tyrius02** [comment]
@conda-forge/core pywin32 itself shouldn't be available for pypy on windows, but apparently it is. Will adding a skip: true w/ a selector to this recipe work?
`
build:
  skip: true  # [win and pypy]
`

or do I need to do something else?

**Comment by tyrius02** [comment]
See PR #64

**Comment by hmaarrfk** [comment]
do you want pypy on linux?

**Comment by tyrius02** [comment]
Though I've only used this package on Linux with PyTorch and never with pypy, I don't see why not.

**Comment by tyrius02** [comment]
Yes, I think this is the right answer. Just didn't know if pypy (or something else) could be used as a selector 

Thanks for the input. Time to go look at pywin32.
